src/ReadTransaction.ts

```diff
@@ -11,49 +14,52 @@ import { Transaction } from './Transaction'
 export class ReadTransaction extends Transaction<ReadTransactionRequest, ReadTransactionReply> {
   public async start() {
     const startReq = new StartTransactionRequest()
-    startReq.setStoreid(super.storeID)
-    startReq.setModelname(super.modelName)
+    startReq.setStoreid(this.storeID)
+    startReq.setModelname(this.modelName)
     const req = new ReadTransactionRequest()
     req.setStarttransactionrequest(startReq)
-    super.client.start()
-    super.client.send(req)
+    this.client.start()
+    this.client.send(req)
   }
 
   public async has(entityIDs: string[]) {
-    return new Promise<boolean>((resolve, reject) => {
+    return new Promise<ModelHasReply.AsObject>((resolve, reject) => {
```

Comment:
These methods now all return their client-level counterpart responses.

Reply:
+1 for the consistency, but some of these are annoying to parse (containing raw JSON) and have non-intuitive param names. It would be nice to parse these out here and return a normal type.

Reply:
Oh sweet, yes let's defs do this! I'll do that in a separate PR tho to keep this one from growing too much... seem reasonable?